test/unit/logger/easy_logging_starter_test.js

```diff
@@ -107,19 +108,35 @@ describe('Easy logging starter tests', function () {
       });
   });
 
-  async function createConfigFile(logLevel, configDirectory, configFileName) {
+  it('should fail for inaccessible log path', async function () {
+    // given
+    const logLevel = 'ERROR';
+    const configFilePath = await createConfigFile(logLevel, tempDir, defaultConfigName, '/?inaccessible');
+
+    // expect
+    await assert.rejects(
+      async () => await init(configFilePath),
+      (err) => {
+        assert.strictEqual(err.name, 'EasyLoggingError');
+        assert.strictEqual(err.message, 'Failed to initialize easy logging');
+        assert.match(err.cause.message, /no such file or directory|permission denied/);
+        return true;
+      });
+  });
+
+  async function createConfigFile(logLevel, configDirectory, configFileName, logPath) {
     const configFilePath = path.join(configDirectory, configFileName);
     const configContent = `{
               "common": {
                   "log_level": "${logLevel}",
-                  "log_path": "${tempDir.replace(/\\/g, '\\\\')}"
+                  "log_path": "${exists(logPath) ? logPath : tempDir.replace(/\\/g, '\\\\')}"
               } 
           }`;
     await writeFile(configFilePath, configContent);
     return configFilePath;
   }
 
   async function writeFile(filePath, fileContent) {
-    await fsPromises.writeFile(filePath, fileContent, { encoding: 'utf8' });
+    await fsPromises.writeFile(filePath, fileContent, { encoding: 'utf8', mode: 0o700 });
```

Comment:
why not 600?

Reply:
This is just a test case, it does not matter if it is 600 or 700. I can change it to 600 if you prefer that.